pkg/operator/starter.go

```diff
@@ -187,7 +189,17 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		ctx.EventRecorder.WithComponentSuffix("cert-rotation-controller"),
 	)
 
-	configmetrics.Register(configInformers)
+	// register operator metrics
+	// TODO: Move this to library-go and expose only registry
+	r := metrics.NewKubeRegistry()
+	// register process start time
+	if err := metrics.RegisterProcessStartTime(r.RawRegister); err != nil {
+		return err
+	}
+	// register config metrics
+	configmetrics.NewRegistry(r).Register(configInformers)
+	// add metrics endpoint
+	ctx.Server.Handler.GoRestfulContainer.Handle("/metrics/operator/v1alpha1", promhttp.HandlerFor(r, promhttp.HandlerOpts{ErrorHandling: promhttp.ContinueOnError}))
```

Comment:
note, @smarterclayton does not like this and we will end up with 2 endpoints and our service monitor will suck because we will need relabeling :-(